sklearn/semi_supervised/_label_propagation.py

```diff
@@ -255,8 +255,12 @@ def fit(self, X, y):
         self : object
             Returns the instance itself.
         """
-        self._validate_params()
-        X, y = self._validate_data(X, y)
+        X, y = self._validate_data(
```

Comment:
I think this line was removed when updating the PR. It will fix failing tests on the CI.
```suggestion
        self._validate_params()

        X, y = self._validate_data(
```

Reply:
done. 

In this pr, the `test_label_propagation.py` takes around 0.75s vs ~0.55s of `main`. (in my laptop)